mock/src/main/java/com/tngtech/keycloakmock/impl/handler/TokenRoute.java

```diff
@@ -111,6 +116,31 @@ private void handlePasswordFlow(RoutingContext routingContext) {
         .end(toTokenResponse(token, session.getSessionId()));
   }
 
+  private void handleClientCredentialsFlow(RoutingContext routingContext) {
+    final String basicAuthToken = routingContext.request().getHeader(HttpHeaderNames.AUTHORIZATION);
+    if (basicAuthToken == null || basicAuthToken.isEmpty()) {
+      routingContext.fail(400);
+      return;
+    }
+
+    final Map<String, String> clientCredentials = new BasicTokenHelper().parseToken(basicAuthToken);
```

Comment:
Please re-use the mechanism that the password flow uses (i.e. take the username and password from the routingContext.user()).